### DISCUSSION TITLE
Normalization is not working, need fixing?

### DISCUSSION THREAD

**Discussion by neurlang** [coqui-ai/TTS#1593] [General Q&A]
Anyone else hear just beep when vocode anything? But when I comment out these lines in synthesizer.py I can hear speech:
`290                    # renormalize spectrogram based on vocoder config
291                    vocoder_input = self.vocoder_ap.normalize(mel_postnet_spec.T)
`
Maybe my normalize factors weren't preset. Another question, taking a look in audio.py, is it normal that it normalizes in the melspectrogram final step? this is used by hifigan. And what does that air breathing removal does there?

Correct me if I'm wrong, but shouldn't the vocoder model (say hifigan) just learn the actual spectrogram (without normalization) and store the training data average of medians (or 5%,95% quantiles) separately in model.json.

When there are average of medians (maybe even for every mel), it will be able to normalize better on inference?

**Comment by erogol** [comment]
You can disable normalization in the config if it is what you want.

**Reply by neurlang** [reply]
yeah but then I need to normalize on my own and the quality is lower than [Griffin-Lim](https://paperswithcode.com/method/griffin-lim-algorithm)
